packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -1094,7 +1096,7 @@ export class AutoScalingGroup extends AutoScalingGroupBase implements
    * - Update the instance's CreationPolicy to wait for `cfn-init` to finish
    *   before reporting success.
    */
-  private applyCloudFormationInit(init: ec2.CloudFormationInit, options: ApplyCloudFormationInitOptions = {}) {
+  public applyCloudFormationInit(init: ec2.CloudFormationInit, options: ApplyCloudFormationInitOptions = {}) {
```

Comment:
One question now that this is public -- what's the behavior if I supply one init in the constructor and then call this separately? (Or call this twice, with either different or the same `init`)? Should we error in that case?

Reply:
I wanna say they just shouldn't do that, but I guess we can add a check on it, sure.